### PR TITLE
SW-5026 Admin -> Console, move link to top bar

### PR DESCRIPTION
**Pull request by nickgraz**
- Move link to accelerator to top bar breadcrumbs
- Rename 'Participants' to 'Accelerator'
- Rename 'Accelerator Admin' to 'Accelerator Console'
- Remove 'accelerator admin' from strings
- Leaving 'Participants' in strings because we are using it later

### CODE REVIEW

**Comment by nickgraz** [comment]
* **#2301** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2301?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/strings/csv/en.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 key_name,en,comment
-ACCELERATOR_ADMIN,Accelerator Admin,
```
Comment (karthikbtf) [inline src/strings/csv/en.csv:2]: this might get added back unless we go into phrase and remove it

**Review by karthikbtf — APPROVED** [review]
This looks much nicer 👍 
Just calling out that these changes won't show up on mobile view. Not sure what the plan is there.

**Comment by nickgraz** [comment]
> This looks much nicer 👍 
> Just calling out that these changes won't show up on mobile view. Not sure what the plan is there.
> 
> this might get added back unless we go into phrase and remove it

If I recall correctly I think the console is only available on desktop, but I'll double check.

**Comment by karthikbtf** [comment]
> > This looks much nicer 👍
> > Just calling out that these changes won't show up on mobile view. Not sure what the plan is there.
> > this might get added back unless we go into phrase and remove it
> 
> If I recall correctly I think the console is only available on desktop, but I'll double check.

I added a mobile view page message for deliverables only, we can move that out into console for sure.

**Comment by nickgraz** [comment]
https://terraformationteam.slack.com/archives/C01VAAP3K0W/p1710354281491249

Looks like we are going to (with minimal effort right now) support mobile. I'll make it visible, thanks for the catch.

File: src/scenes/OrgRouter/NavBar.tsx [hunk 2]
```diff
@@ -175,12 +175,12 @@ export default function NavBar({
       setShowNavBar={setShowNavBar as React.Dispatch<React.SetStateAction<boolean>>}
       backgroundTransparent={backgroundTransparent}
     >
-      {featureFlagAccelerator && user && (
+      {isMobile && featureFlagAccelerator && user && (
```
Comment (karthikbtf) [inline src/scenes/OrgRouter/NavBar.tsx:178]: `isAllowedViewConsole` check?

**Review by karthikbtf — APPROVED** [review]
lgtm, one question

File: src/scenes/OrgRouter/NavBar.tsx (hunk 2 above)
Comment (nickgraz) [reply src/scenes/OrgRouter/NavBar.tsx:178]: Ah yes, good catch.

**Review by karthikbtf — APPROVED** [review]
👍

**Pull request merged**